particles/Tutorial/Javascript/Demo/source/MoveApplier.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
MoveApplier does more than apply moves (specifically, it determines end conditions). This seems like Game to me.